src/scripts/hooks/utils/cycle.js

```diff
@@ -104,10 +102,16 @@ module.exports = {
 
           nu = {};
```

Comment:
what stands 'nu' for?

Reply:
don't know why :)  Code comes from here - https://github.com/douglascrockford/JSON-js/blob/master/cycle.js, and this is also referred to in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Cyclic_object_value